svthalia/concrexit
# Promo Request status updates for organisers

Closes #3005 .
Closes #3483.



### Summary

An update email will be sent out to the organising committees when the status of a promotion request is updated. There will also be a daily summary email sent out to the promocie with all updates.

### How to test

1. Create promotion request for an event on the admin page.

   i. Confirm no email sent at this point. (in terminal)

2. Change the status of the just created promotion on the promocie admin page.

   i. Confirm email sent to all organising committees (in terminal)

3. open a `manage.py shell` instance, and run the following:
    ```python
    from promotion.tasks import promo_update_overview_daily
    
    promo_update_overview_daily()
    ```

   i. Confirm that the daily update overview was created for the promocie.

4. In the `manage.py shell` run the following:
    ```python
    from promotion.models import PromotionRequest
    
    PromotionRequest.objects.filter(status_updated=True)
    ```

   i. Confirm this returns an empty QuerySet
